### PR TITLE
feat(navbar): align navbars to design system

### LINKED ISSUE #548: [Design System] Navigation bars
**Work required**
- Align navigation bar design with design system
- Decompose and introduce custom components for code sharing across navbar variants as necessary
- Ensure following navigation bars are reviewed:
   - Landing page navbar 
   - Dashboard navbar
   - Builder navbar
   - Preview navbar

Refer to [CheckFirst Design System](https://www.figma.com/file/X3oi4JnynZuJ2j6zSHDnTH/CheckFirst-Design-System) for details.

**Example**
image

image

### PR DESCRIPTION
**Pull request by orbitalsqwib**
## Problem

Align navbars to the design system via custom components.

Closes #548

## Solution

**Features**:

- Implement custom styled tab components with theme styles

**Improvements**:

- simplify layout
- update styles to align with design system
- minor updates to copy

## Before & After Screenshots

**BEFORE**:
Screenshot 2021-06-11 at 3 45 13 PM
Screenshot 2021-06-11 at 3 45 23 PM
Screenshot 2021-06-11 at 3 45 04 PM
Screenshot 2021-06-11 at 3 44 55 PM

**AFTER**:
Screenshot 2021-06-11 at 3 44 17 PM
Screenshot 2021-06-11 at 3 44 24 PM
Screenshot 2021-06-11 at 3 44 03 PM
Screenshot 2021-06-11 at 4 16 20 PM

## Tests

- [ ] Check that design aligns to design system
- [ ] Standard regression tests
  - [ ] Create a new checker from scratch
  - [ ] Create a new checker from template
  - [ ] Save draft and preview checker
  - [ ] Publish checker
  - [ ] Verify live checkers

### CODE REVIEW

File: src/client/pages/FormBuilder.tsx [hunk 1]
```diff
@@ -19,13 +19,7 @@ const WithNavBar: FC = ({ children }) => (
...
     <Navbar />
     <HelpButton />
-    <Container
-      h="calc(100vh - 80px)"
-      overflow="auto"
-      mt="80px"
-      maxW="100vw"
-      pb="32px"
-    >
+    <Container h="calc(100vh - 73px)" overflow="auto" mt="73px" maxW="100vw">
```
Comment (orbitalsqwib) [inline src/client/pages/FormBuilder.tsx]:
```suggestion
    <Container
      h="calc(100vh - 73px)"
      overflow="auto"
      mt="73px"
      maxW="100vw
      pb="32px"
    >
```
Hey Kee Wei, I think you might have accidentally undone the bottom padding fix here

File: src/client/pages/FormBuilder.tsx [hunk 2]
```diff
@@ -36,13 +30,7 @@ const WithNavBar: FC = ({ children }) => (
...
   <Flex direction="column" minH="100vh" bgColor="neutral.200">
     <PreviewNavBar />
-    <Container
-      h="calc(100vh - 80px)"
-      overflow="auto"
-      mt="80px"
-      maxW="100vw"
-      pb="32px"
-    >
+    <Container h="calc(100vh - 73px)" overflow="auto" mt="73px" maxW="100vw">
```
Comment (orbitalsqwib) [inline src/client/pages/FormBuilder.tsx]:
```suggestion
    <Container
      h="calc(100vh - 73px)"
      overflow="auto"
      mt="73px"
      maxW="100vw
      pb="32px"
    >
```
Same as above

**Comment by orbitalsqwib** [comment]
Note that this should also partially solve #393

**Review by lamkeewei — APPROVED** [review]
lgtm!

**Pull request merged**